Dockerfile

```diff
@@ -20,5 +24,4 @@ WORKDIR /usr/local/bin
 
 HEALTHCHECK CMD exit 0
 
-ENTRYPOINT [ "/usr/bin/dumb-init", "docker-bench-security.sh" ]
-
+ENTRYPOINT [ "/usr/bin/dumb-init", "get-specific-version.sh", "docker-bench-security.sh" ]
```

Comment:
Not a big fan about adding this to the `ENTRYPOINT`, `get-specific-version` should be a function in `helper_lib.sh`.